client/landing/gutenboarding/components/plans/plans-details/index.tsx

```diff
@@ -18,50 +18,54 @@ const PlansDetails: React.FunctionComponent = () => {
 	const { __ } = useI18n();
 
 	return (
-		<table className="plans-details">
-			<thead>
-				<tr className="plans-details__header-row">
-					<th>{ __( 'Feature' ) }</th>
-					<th>{ __( 'Free' ) }</th>
-					<th>{ __( 'Personal' ) }</th>
-					<th>{ __( 'Premium' ) }</th>
-					<th>{ __( 'Business' ) }</th>
-					<th>{ __( 'Commerce' ) }</th>
-				</tr>
-			</thead>
-			{ plansDetails.map( ( detail ) => (
-				<tbody key={ detail.id }>
-					{ detail.name && (
+		<div className="plans-details">
+			<div className="plans-details__viewport">
+				<table className="plans-details__table">
+					<thead>
 						<tr className="plans-details__header-row">
-							<th colSpan={ 6 }>{ detail.name }</th>
+							<th>{ __( 'Feature' ) }</th>
+							<th>{ __( 'Free' ) }</th>
+							<th>{ __( 'Personal' ) }</th>
+							<th>{ __( 'Premium' ) }</th>
+							<th>{ __( 'Business' ) }</th>
+							<th>{ __( 'Commerce' ) }</th>
```

Comment:
BTW we might need to pull these names from constants at some point, but we can do that in future PRs.